DigitalLearningSolutions.Web/Styles/frameworks/frameworksShared.scss

```diff
@@ -1,274 +1,274 @@
-﻿@import "~nhsuk-frontend/packages/core/all";
-@import "../shared/headingButtons";
-@import "../shared/searchableElements/searchableElements";
-$nhs-dark-grey: #425563;
+﻿@use "nhsuk-frontend/packages/core/all" as *;
+@use "../shared/headingButtons" as *;
+@use "../shared/searchableElements/searchableElements" as *;

```

Comment:
These are the only real changes on this file, the other "changes" are just line endings being corrected to CRLF.